### PR TITLE
OpenBabelReader and Tests

### PR DESCRIPTION
**Pull request by lunamorrow**
In reference to #5. This is a draft of the code with the initial functionality starting to be roughly scaffolded out for feedback.

PR Checklist
------------
 - [X] Tests?
 - [ ] Docs?
 - [ ] CHANGELOG updated?
 - [X] Issue raised/referenced?

### CODE REVIEW

File: mda_openbabel_converter/tests/test_openbabel_reader.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    def test_coordinates(self, n_frames):
+
+        obconversion = ob.OBConversion()
+        obconversion.SetInFormat("pdb")
+        obmol = ob.OBMol()
+        obconversion.ReadFile(obmol, "mda_openbabel_converter/data/1crn.pdb")
+        universe = mda.Universe(obmol)
+        assert universe.trajectory.n_frames == n_frames
+        expected = np.array([
+            [atom.GetVector().GetX(),
```
Comment (xhgchen) [inline mda_openbabel_converter/tests/test_openbabel_reader.py:55]: Cédric's walrus operator suggestion looks applicable here as well.

**Comment by xhgchen** [comment]
Great work! Loving the progress :)

File: mda_openbabel_converter/OpenBabel.py [hunk 2]
```diff
@@ -6,35 +6,57 @@
...
+            return isinstance(thing, OBMol)
 
     def __init__(self, filename: OBMol, **kwargs):
         """
         Converts file to OBMol to AtomGroup
         """
-        pass
+        n_atoms = filename.NumAtoms()
+        coordinates = np.array([
+            [atom.GetVector().GetX(),
```
[the remark this answers is not in the document]
Comment (lunamorrow) [reply mda_openbabel_converter/OpenBabel.py:51]: Oh good idea, that'll save some time. Thanks @exs-cbouy

File: mda_openbabel_converter/tests/test_openbabel_reader.py (hunk 1 above)
Comment (lunamorrow) [reply mda_openbabel_converter/tests/test_openbabel_reader.py:55]: Good point!

**Comment by lunamorrow** [comment]
> Could you also add a test for mols without any coordinates making sure that they are all `nan`?

@exs-cbouy It looks like null position in OpenBabel is stored as a (0, 0, 0) vector. I don't want to report that as a position when its not, so I'm thinking that I can check that if the whole molecules position vectors are (0, 0, 0) that there is no position info and to set it to null (or not even set it) and issue a warning. That will prevent molecules with an atom at a valid (0, 0, 0) position from being flagged. Thoughts?

**Review by hmacdope — APPROVED** [review]
LGTM

**Pull request merged**